pkg/controller/scheduledjob/utils.go

```diff
@@ -125,7 +125,11 @@ func getNextStartTimeAfter(schedule string, now time.Time) (time.Time, error) {
 // If there were missed times prior to the last known start time, then those are not returned.
 func getRecentUnmetScheduleTimes(sj batch.ScheduledJob, now time.Time) ([]time.Time, error) {
 	starts := []time.Time{}
-	sched, err := cron.Parse(sj.Spec.Schedule)
+	tmpSched := sj.Spec.Schedule
```

Comment:
This looks like only parsed data adjust, so maybe it should have other name, but still extracting this and adding a name for this block - could improve readability of this code.